### DISCUSSION TITLE
Code Review: Show only my owned code

### DISCUSSION THREAD

**Discussion by Cossey** [community/community#40962] [Mobile]
I would like to review pull requests on mobile - however the ones of relevance are covering a large part of the project (openhab/openhab-addons) whereas I only have ownership of a small portion of the code base needing review. 

Add a filter in the files section in the pr for "show only files you own". Rarely do I want to review approve other files in these larger PR and looking through 100's of files is tedious on iphone. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.91.0 (85250348)
Device: iPhone Xs (iOS 16.1.1)
</details>

**Comment by ossanna16** [comment]
Hi there @Cossey and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
